variables.tf

```diff
@@ -218,3 +218,33 @@ variable "alert_region" {
   type        = string
   default     = "us-east-2"
 }
+
+variable "max_capacity" {
+  description = "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/appautoscaling_target#max_capacity"
+  type        = number
+  default     = 1
```

Comment:
I think we could go ahead and take this to 10 or so by default, then min_capacity is all that would need to be updated in a production situation